src/main/java/at/hannibal2/skyhanni/utils/DelayedRun.kt

```diff
@@ -1,34 +1,24 @@
 package at.hannibal2.skyhanni.utils
 
-import at.hannibal2.skyhanni.utils.LorenzUtils.editCopy
+import java.util.concurrent.LinkedBlockingQueue
 import kotlin.time.Duration
 
-// TODO find better sync bug fix than creating a new map for each use
 object DelayedRun {
-    var map = mapOf<() -> Any, SimpleTimeMark>()
+    val map = mutableListOf<Pair<() -> Any, SimpleTimeMark>>()
```

Comment:
The naming of both of these variables could use some improvement: Maybe `tasks` and `delayedTasks`.